events/2024.11.Munich/Readme.md

```diff
@@ -39,6 +39,7 @@ Stakeholders from related SDOs are welcome to contribute.
 | TU Munich    | Hololens                                      |  x       |        |         | HTTP               | Wifi                                                               | no sec should be used; QR code to TD is needed      |   tbc |
 | UC Dublins   | ASTRA/MAMS Platform                           |  X       |        |         |                    |                                                                    |  Multi agent system platform (only software)        | https://github.com/RemCollier |
 | Uni St. Gallen |  YggDrasil                                  |          |        |  X      | HTTP               | ...                                                                |  ...                                                | https://github.com/jeremylemee |
+| Invited Experts | Cross-platform app (using `dart_wot`)      | x        |        |  X      | HTTP, CoAP, MQTT   | WiFi                                                               |  ...                                                | @JKRhb |
```

Comment:
```suggestion
| Eclipse Thingweb | Cross-platform app (using `dart_wot`)      | x        |        |  X      | HTTP, CoAP, MQTT   | WiFi                                                               |  ...                                                | @JKRhb |
```

I was going to add the other thingweb stuff and thought it is best to put Thingweb as the "owner" of the entities.

Reply:
Thank you, I was about to ask you about that column :)